### ISSUE TITLE
Name of material

### ISSUE THREAD

**Issue by arvifox** [google-ar/sceneform-android-sdk#315]
Hi everyone.
I'd like to give users a way to change a color and texture of material.
Renderable has `getMaterial()` method, but `Material` class has no name.
Now I display a spinner to select material. It contains just a index. It's not user friendly way.
It would be great if material has name.

**Comment by gstanlo** [comment]
I don't believe material names are kept in Sceneform bundle files. Can you explain your process some more? What type of file are they being stored in?

**Comment by arvifox** [comment]
There is a name of material in sfa files.
![img-2018-09-12-13-41-41](https://user-images.githubusercontent.com/7602319/45420043-c2d43380-b691-11e8-9795-6e1c3781e9fb.png)
I'd like to have access to these names in my code.

**Comment by romainguy** [comment]
Custom materials also support setting a name. It's just a matter of exposing those names through public APIs in Sceneform.

**Comment by gstanlo** [comment]
To clarify, is it that you would like to have the friendly name of a material accessible? That is, you might have multiple instances of the same (possibly custom) material with different parameters, and you want the names of those -- not the name of the custom or sfm material -- correct?

**Comment by arvifox** [comment]
yes, correct.
